platforms/software/build-init/src/main/java/org/gradle/buildinit/plugins/internal/GitAttributesGenerator.java

```diff
@@ -30,13 +30,17 @@ public void generate(InitSettings settings, BuildContentGenerationContext buildC
         File file = settings.getTarget().file(".gitattributes").getAsFile();
         try {
             try (PrintWriter writer = new PrintWriter(new FileWriter(file))) {
-                writer.println("#");
-                writer.println("# https://help.github.com/articles/dealing-with-line-endings/");
-                writer.println("#");
-                writer.println("# Linux start script should use lf");
+                if (settings.isWithComments()) {
```

Comment:
🤔 These comments are less intrusive IMO and potentially more useful as a user opens this file rarely. I would consider keeping them in always.

Reply:
I wanted to remove these comments as well, for consistency, but I agree, these are not very intrusive.
As I don't have a strong opinion here, I'll keep these comments in.

Reply:
Done